antora-ui-camel/src/css/blog.css

```diff
@@ -106,6 +106,14 @@ article.blog p {
   margin-bottom: 1rem;
 }
 
+.blog .post .post-content .table-wrapper {
+  overflow-x: auto;
+}
+
+.blog .post .post-content .link-wrapper {
```

Comment:
Same remark as the `table-wrapper`. This is bringing on future complexity we can't maintain.

Reply:
I found a way across this to not make use of link-wrapper.